truffle/test/marketplace_test.js

```diff
@@ -1,255 +1,378 @@
 /* -------------------------------------------------------------------------- */
 
 const TuiChainController = artifacts.require("TuiChainController");
-const TuiChainMarket     = artifacts.require("TuiChainMarket");
-const TuiChainLoan       = artifacts.require("TuiChainLoan");
-const TuiChainToken      = artifacts.require("TuiChainToken");
```

Comment:
Align the assignments. It's good pratice

Reply:
AFAICT, Prettier unfortunately has no option for this. Although I also prefer to align assignments, I'd argue that it's more a matter of taste than of good practice.